Packs/AutofocusFeed/Integrations/AutofocusFeed/AutofocusFeed.yml

```diff
@@ -0,0 +1,122 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: AutoFocus Feed
+  version: -1
+configuration:
+- display: Indicator feed
+  name: indicator_feeds
+  options:
+  - Daily Threat Feed
+  - URL Feed
+  required: true
+  type: 16
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- display: The AutoFocus API key
+  name: api_key
+  required: true
+  type: 4
+- display: The URL feed ID to fetch
+  name: output_feed_id
+  required: false
+  type: 0
+- display: The name of the feed ID to fetch
+  name: output_feed_name
+  required: false
+  type: 0
+- display: Bypass exclusion list
+  name: bypassExclusionList
+  defaultvalue: ""
+  type: 8
+  required: false
+  additionalinfo: When selected, the exclusion list will be ignored for this feed.
+- defaultvalue: F - Reliability cannot be judged
+  display: Reliability
+  name: feedInstanceReliability
+  options:
+  - A - Completely reliable
+  - B - Usually reliable
+  - C - Fairly reliable
+  - D - Not usually reliable
+  - E - Unreliable
+  - F - Reliability cannot be judged
+  required: true
+  type: 15
+- name: expiration
+  required: false
+  type: 17
+- defaultvalue: indicatorType
+  name: expirationPolicy
+  required: false
+  type: 0
+- name: expirationInterval
+  required: false
+  type: 1
+- display: Fetch indicators
+  name: feed
+  required: false
+  type: 8
+- display: Bypass exclusion list
+  name: bypassExclusionList
+  required: false
+  type: 8
+- defaultvalue: feedInstanceReputationNotSet
+  display: Mark as reputation
+  name: feedInstanceReputation
+  options:
+  - None
+  - Good
+  - Suspicious
+  - Bad
+  required: false
+  type: 18
+- defaultvalue: '240'
+  display: Fetch Interval
+  name: feedFetchInterval
+  required: false
+  type: 19
+description: Indicators Fetched from AutoFocus Feeds
+display: AutoFocus Feed
+name: AutoFocus Feed
+script:
+  commands:
+  - arguments:
+    - default: false
+      description: How many indicators should be returned
+      isArray: false
+      name: limit
+      required: false
+      secret: false
+    - default: false
+      defaultValue: '0'
+      description: 'The index or the first indicator to fetch '
+      isArray: false
+      name: page
```

Comment:
Perhaps change this to `offset`? I don't think that `page` is the correct terminology here.

Reply:
renamed